src/utils/dynamo.py

```diff
@@ -47,10 +47,6 @@ def write_to_dynamo(
     if isTesting:
         existingItem["country"]["S"] = existingItem["country"]["S"] + testSuffix
 
-    # Mark the existing item as 'old', then scrape to get the most 'up to date' information.
-    if key_exists:
```

Comment:
Given that `key_exists` is no longer used here, should we update surrounding areas, such as L36?

Reply:
Yeah you're right, I'll fix that asap